site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx

```diff
@@ -64,25 +74,28 @@ export interface WorkspaceScheduleFormProps {
 }
 
 export interface WorkspaceScheduleFormValues {
+  autoStartEnabled: boolean
   sunday: boolean
   monday: boolean
   tuesday: boolean
   wednesday: boolean
   thursday: boolean
   friday: boolean
   saturday: boolean
-
   startTime: string
   timezone: string
+
+  autoStopEnabled: boolean
```

Comment:
Sorry, I know we chatted about this yesterday, but why are we including `autoStartEnabled` and `autoStopEnabled` as form values instead of stateful variables? The API doesn't have a concept of these values, right?

Reply:
Right. I just thought it was a little cleaner this way:
- validation can go back to being an object instead of a function
- all fields in the form are handled by the form
- now that I'm persisting pre-submission form state, the `formToRequest` functions need to refer to the enabled values to decide whether to send the backend falsey values

And the `formToRequest` functions just create values to return rather than passing everything through, so there wasn't a real downside to having extra data in the form.

Reply:
Sounds good, thanks!